pkg/reconciler/brokercell/broker_targets_config.go

```diff
@@ -147,12 +147,12 @@ func (r *Reconciler) updateTargetsConfig(ctx context.Context, bc *intv1alpha1.Br
 		UpdateFunc: func(oldObj, newObj interface{}) { r.refreshPodVolume(ctx, bc) },
 		DeleteFunc: nil,
 	}
-	_, err = r.cmRec.ReconcileConfigMap(bc, desired, resources.TargetsConfigMapEqual, handlerFuncs)
+	_, err = r.cmRec.ReconcileConfigMap(ctx, bc, desired, resources.TargetsConfigMapEqual, handlerFuncs)
 	return err
 }
 
 func (r *Reconciler) refreshPodVolume(ctx context.Context, bc *intv1alpha1.BrokerCell) {
-	if err := volume.UpdateVolumeGeneration(r.KubeClientSet, r.podLister, bc.Namespace, resources.CommonLabels(bc.Name)); err != nil {
+	if err := volume.UpdateVolumeGeneration(context.TODO(), r.KubeClientSet, r.podLister, bc.Namespace, resources.CommonLabels(bc.Name)); err != nil {
```

Comment:
can you use the ctx passed as parameter instead of creating a new one

Reply:
done